src/amber/server/env.cr

```diff
@@ -1,33 +1,38 @@
 module Amber
   def self.env
-    Env.new(Settings.env.not_nil!.downcase)
+    @@env ||= Env.new(ENV["AMBER_ENV"]? || "development") # Would be best to only define this once.
   end
 
   class Env
-    def initialize(@env : String | Symbol)
+    alias EnvType = String | Symbol 
+
+    def initialize(@env : String) end
+
+    def is?(environment : EnvType)
+      @env == environment.to_s.downcase
     end
 
-    def in?(environment_list : Array(String | Symbol))
-      (environment_list.map &.to_s.downcase).includes? @env
+    def in?(environment_list : Array(EnvType))
+      environment_list.each do |other_env|
+        return true if is?(other_env)
+      end
     end
 
-    def is?(environment : String | Symbol)
-      @env == environment.to_s.downcase
+    def in?(*environment_list : Object)
+      in?(environment_list.to_a)
     end
 
     def to_s(io)
       io << @env
     end
 
-    private def environments
-      (Dir.entries(CONFIG_DIR).map &.downcase.tr(".yml", "") + ENVIRONMENTS).to_set
+    def ==(other : EnvType)
```

Comment:
Also a nice addition

Reply:
With this I didn't want to change any part of the outside of the blackbox. I'm fine with removing this but sort of thought of it last night as [] and fetch in a hash.
https://github.com/crystal-lang/crystal/blob/d1f8c42f82d3e6491cb7b3b536984c19a51b94ba/src/hash.cr#L59